src/components/atoms/datePicker/datePicker.scss

```
@@ -2,4 +2,15 @@ input[type="date"] {
   padding: $small-space;
   border: 1px solid $gray-light;
   border-radius: $regular-radius;
-}
+
+  &:hover {
+    background-color: $primary-utility-color;
+    border: 1px solid $primary-dark-color;
+    border-radius: $regular-radius;
+  }
+
+  &:focus {
+    border: 2px solid $gray-lighter;
+    border-radius: $regular-radius;
+  }
+}
```

```suggestion
}

```
Manque un retour a la ligne

---

Déjà ajouté